### PR TITLE
Fix filters dark styling

### PR DESCRIPTION
**Pull request by drskullster**
Before

image

After

image

### CODE REVIEW

File: src/components/Filters/index.tsx [hunk 1]
```diff
@@ -147,6 +147,16 @@ export const Filters = ({
 		[views, onViewsChange],
 	);
 
+	// TODO remove when we have implemented a dark theme
+	const negativeStyles: Material.SxProps = {
```
Comment (JSReds) [inline src/components/Filters/index.tsx:151]: same as above

File: src/components/Filters/Views.tsx [hunk 2]
```diff
@@ -72,6 +72,14 @@ export const Views = ({ views, setFilters, deleteView, dark }: ViewsProps) => {
 		});
 	}, [views]);
 
+	const negativeStyles: Material.SxProps = {
```
[the remark this answers is not in the document]
Comment (myarmolinsky) [reply src/components/Filters/Views.tsx:75]: +1

File: src/components/Filters/index.tsx [hunk 3]
```diff
@@ -194,10 +201,12 @@ export const Filters = ({
 				{(!renderMode || renderMode.includes('add')) && (
 					<Button
 						variant={dark ? 'light' : 'outlined'}
-						color="secondary"
 						onClick={() => setShowFilterDialog(true)}
 						startIcon={matches ? <FontAwesomeIcon icon={faFilter} /> : null}
-						sx={{ whiteSpace: 'nowrap' }}
+						sx={{
+							whiteSpace: 'nowrap',
+							...(dark ? darkStyles : undefined),
```
Comment (myarmolinsky) [inline src/components/Filters/index.tsx:208]: nit
```suggestion
							...(dark && darkStyles),
```

File: src/components/Filters/Search.tsx [hunk 4]
```diff
@@ -43,18 +45,38 @@ export const Search = ({
...
+					? {
+							color: 'white',
+							'&::before': {
+								borderBottom: 'solid rgba(255, 255, 255, 0.6) 1px',
+							},
+							'&:hover:not(.Mui-disabled, .Mui-error)::before': {
+								borderBottom: 'solid white 1px',
+							},
+					  }
+					: undefined,
```
Comment (myarmolinsky) [inline src/components/Filters/Search.tsx:79]: nit
```suggestion
				// TODO remove when we have implemented a dark theme
				...(dark && { sx: {
							color: 'white',
							'&::before': {
								borderBottom: 'solid rgba(255, 255, 255, 0.6) 1px',
							},
							'&:hover:not(.Mui-disabled, .Mui-error)::before': {
								borderBottom: 'solid white 1px',
							},
					  }),
```

**Pull request merged**